MODULE.bazel

```diff
@@ -4,6 +4,38 @@ module(
     compatibility_level = 0,
 )
 
+
+http_archive = use_repo_rule("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+
+# Sysroot for cross-compilation
+http_archive(
+    name = "sysroot_linux_aarch64",
+    build_file_content = """
+filegroup(
+  name = "sysroot",
+  srcs = glob(["*/**"]),
+  visibility = ["//visibility:public"],
+)
+""",
+    sha256 = "8462b3c1bbb7d6e6b5ec48e3b64d59671155b0a522a28e3bcabef67bced3e008",
+    urls = ["https://github.com/scasagrande/toolchains_llvm_sysroot/releases/download/linux-sysroot-4_18-2_28-10_3_0/linux-sysroot-aarch64.tar.zst"],
+)
```

Comment:
can we move this down the have the genera common MODULE file layout of `bazel_dep` calls, extensions calls, other `http_archvie` calls?